bin/fetch_configlet.dart

```diff
@@ -0,0 +1,120 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:archive/archive.dart';
+
+Future<Null> main() async {
+  exitCode = await fetch_configlet().catchError((dynamic onError) {
+    print('Failed with error:\n$onError');
+    return 1;
+  });
+
+  exit(exitCode);
+}
+
+final githubApi = Uri.parse('https://api.github.com/repos/exercism/configlet/releases?per_page=1');
```

Comment:
Recommend moving this global variable above the `main` function. And changing it to a private variable.